interfaces/index.ts

```diff
@@ -1,5 +1,5 @@
 export type ResponseType = {
-  data: [] | {} | null;
+  data: [] | null | void;
```

Comment:
why void?

Reply:
because we are returning a void data  in new_data

Reply:
data should not return void in success cases. it should return something right

Reply:
but it's void type in our code

Reply:
could you please share the code which is void type in our code

Reply:
![image](https://user-images.githubusercontent.com/66621672/189536112-ed486289-065a-4722-a7a4-d3064343df84.png)
new_user is of void type

Reply:
you shouldn't directly return the new_user like this. see the response you are getting from the create function. and put that response inside the data. and also please make the variable name in camelCase.